ansible_navigator/actions/inventory.py

```diff
@@ -455,9 +456,13 @@ def _collect_inventory_details(self) -> None:
                     inventory_output = "{" + parts[1]
                 else:
                     inventory_output = ""
-            if inventory_err:
-                msg = f"Error occurred while fetching ansible inventory: '{inventory_err}'"
-                self._logger.error(msg)
+            warn_msg = ["Errors were encountered while gathering the inventory:"]
+            warn_msg += inventory_err.splitlines()
+            self._logger.error(" ".join(warn_msg))
+            if "Error" in inventory_err:
```

Comment:
```suggestion
            if "ERROR!" in inventory_err:
```
Here's what I was testing with:
```
$ cat invalid.yml
plugin: notinstalled
```

```
$ ANSIBLE_INVENTORY_ANY_UNPARSED_IS_FAILED=True ansible-navigator inventory -i invalid.yml --graph --ee false
```
(no output)

Reply:
I didn't see ERROR! when I recreated the error with an invalid ansible.cfg, I think it might be being removed back in runner

Reply:
![image](https://user-images.githubusercontent.com/18386516/119908513-9e831880-bf07-11eb-816f-5331a97ccf8f.png)

Reply:
(btw, the dupe is a known issue & logged)